src/app/public/plugin-resources/code-examples/split-view/page-bound/split-view-page-bound-demo.component.scss

```diff
@@ -0,0 +1,37 @@
+// START PAGE FLEX
+.page-flex {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
+.page-flex-header {
+  background-color: #fff;
+  border-bottom: 1px solid #ccc;
```

Comment:
Probably want to use approved SCSS variables if we expect people to copy/paste. Better yet, you might be able to use the CSS classes found in our docs: https://developer.blackbaud.com/skyux/design/styles/borders

Reply:
These actually weren't needed anymore after switching to the page summary. I have removed them.